src/components/DropdownMenu.jsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import BaseButton from "@/components/common/BaseButton";
+import MoreIcon from "@assets/icons/more";
+
+const DROPDOWN_LIST_POSITION = {
+  left: "right: 0; left: auto;",
+  right: "left: 0; right: auto;",
+};
+
+const DropdownMenu = ({
+  dropdownItem1,
+  onDropdownItem1Click,
+  dropdownItem2,
+  onDropdownItem2Click,
+  position = "left",
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleToggle = () => setIsOpen((prev) => !prev);
```

Comment:
💬 여담
reducer 를 사용해 아래처럼 간단하게도 작성 가능합니다~

```suggestion
const [isOpen, toggleIsOpen] = useReducer((prev) => !prev, false);
```